lib/reporting/drop_off_report.rb

```diff
@@ -345,11 +371,13 @@ def idv_doc_auth_phone_visited
     end
 
     def idv_enter_password_visited
-      data[Events::IDV_ENTER_PASSWORD_VISITED].count
+      (data[Events::IDV_ENTER_PASSWORD_VISITED] &&
+        data[Events::OLD_IDV_ENTER_PASSWORD_VISITED]).count
```

Comment:
this should be a  `|` or `+` to combine the sets of UUIDs from both right?
```suggestion
      (data[Events::IDV_ENTER_PASSWORD_VISITED] +
        data[Events::OLD_IDV_ENTER_PASSWORD_VISITED]).count
```